avro/profile.json

```diff
@@ -0,0 +1,160 @@
+{
+  "name":"profile",
+  "type":"record",
+  "namespace": "dto",
+  "doc":"Schema for cosmos api container 'profiles'",
+  "fields":[
+    {
+      "name": "id",
+      "type": "string"
+    },
+    {
+      "name":"version",
+      "type":"int",
+      "default":0
+    },
+    {
+      "name":"acceptedTosVersion",
+      "type": "int",
+      "default": "0"
+    },
+    {
+      "name":"blockedInboxOrChannels",
+      "type":{
+        "type": "map",
+        "values":{
+          "type": "array",
+          "items": {
+            "type":"enum",
+            "name":"BlockedInboxOrChannelEnum",
+            "symbols":[
+              "EMAIL",
+              "WEBHOOK",
+              "INBOX"
+            ]
+          }
+        }
+      },
+      "default" : {}
+    },
+    {
+      "name": "email",
+      "type": "string"
+    },
+    {
+      "name":"fiscalCode",
+      "type": "string"
+    },
+    {
+      "name": "isEmailEnabled",
+      "type": "boolean"
+    },
+    {
+      "name": "isEmailValidated",
+      "type": "boolean"
+    },
+    {
+      "name": "isInboxEnabled",
+      "type": "boolean"
+    },
+    {
+      "name": "isWebhookEnabled",
+      "type": "boolean"
+    },
+    {
+      "name": "isTestProfile",
+      "type": ["null","boolean"],
+      "default": null
+    },
+    {
+      "name":"lastAppVersion",
+      "type": ["null","string"],
+      "default":null
+    },
+    {
+      "name": "preferredLanguages",
+      "type":["null", {
+        "type": "array",
+        "items": {
+          "name": "PreferredLanguage",
+          "type":"enum",
+          "symbols":[
+            "it_IT",
+            "en_GB",
+            "es_ES",
+            "de_DE",
+            "fr_FR"
+          ]
+        }
+      }],
+      "default": null
+    },
+    {
+      "name":"pushNotificationsContentType",
+      "type":["null",{
+        "name":"PushNotificationsContentTypeEnum",
+        "type":"enum",
+        "symbols":[
+          "UNSET",
+          "FULL",
+          "ANONYMOUS"
+        ]
+      }],
+      "default":null
+    },
+    {
+      "name": "reminderStatus",
+      "type": ["null",{
+        "type":"enum",
+        "name":"ReminderStatus",
+        "symbols":[
+          "ENABLED",
+          "DISABLED"
+        ]
+      }],
+      "default":null
+    },
+    {
+      "name":"servicePreferencesSettings",
+      "type": [{
+        "name":"ServicesPreferencesMode",
+        "type": "record",
+        "fields":[
+          { 
+            "name":"mode",
+            "type":{
+              "type":"enum",
+              "name":"ServicesPreferencesModeEnum",
+              "symbols":["AUTO","MANUAL"]
+            }
+          },
+          {
+            "name":"version",
+            "doc": "This value can not be negative. Avro schema does not support unsigned integers",
+            "type":"int",
+            "default": 0
+          }
+        ]
+      },{
+        "name":"ServicesPreferencesModeLegacy",
```

Comment:
Since both types will be generated with `version` of type `number`, I'm not sure this definition is necessary.

Reply:
I included a different type to enforce the difference at type level, do you think flatten this type is better?

Reply:
I think that for this purpose minimal type is more manageable.

Reply:
edited in https://github.com/pagopa/io-functions-elt/pull/81/commits/88c71aa7bc4e6273ef4ad4f96ce1002b179bbf6e